docs/application/dotnet/get-started/index.md

```diff
@@ -4,65 +4,27 @@ You can create the Tizen .NET applications in C#. You can also use the Tizen .NE
 
 ## Environment configuration
```

Comment:
@sagarkedari  I don't understand this request. This file was used as a base but now we have NUI based first app and we don't want to bother 3rd party developers about specific profiles.

Reply:
@mihashco, Since this is an old PR, the base file that you had used (index.md) was old and you made changes to it. Since we had released Tizen 6.5 version recently, the base file has been updated (please see latest content of the file here : https://docs.tizen.org/application/dotnet/get-started/). So, I am requesting to use the latest **index.md** file as base again and incorporate your changes to it. By doing this way, we will not miss your changes as well as the latest Tizen related changes in this file. I hope am clear now.